src/features/chainlink-automation/data/chainlink-automation-config.json

```diff
@@ -321,7 +321,7 @@
     "maxPerformDataSize": 2000,
     "flatFeeMicroLink": 0,
     "stalenessSeconds": 90000,
-    "registrar": "0x1433C1BDfCF1b2E1CE843Ede6FcC5d6e409fE56E",
+    "registrar": "0xaB9718E1DC7BD61Bd3440Fb771CC23DA0C0181a1",
```

Comment:
Should this be the registrar address `0xA6deE996014d9670144943c2bc90755829398d07` as shown in your change to [chainlink-automation-addresses.json](https://github.com/smartcontractkit/documentation/pull/1647/files#diff-c5baa47f887aec758a3420ae29244422e1f111a0178aa9fc8b0ba2381ffcf5ff)?